3_optimize_ws_goroutines/kqueue_darwin.go

```diff
@@ -0,0 +1,101 @@
+// +build darwin
+
+package main
+
+import (
+	"log"
+	"reflect"
+	"sync"
+	"syscall"
+
+	"github.com/gorilla/websocket"
+)
+
+type epoll struct {
```

Comment:
can you rename the struct? `kqueue` is maybe a good pick

Reply:
Because main.go declare `*epoll`, maybe I rename to a normal name, eg `event`,

Reply:
Sure, whatever you find that fits, just general enough to make it clear for both epoll and kqueue
also please make every change the same for both the 3rd and 4th examples